onchain/rollups/contracts/history/IHistory.sol

```diff
@@ -20,7 +20,12 @@ interface IHistory {
     ///         The encoding of `_claimData` might vary
     ///         depending on the history implementation.
     /// @param _claimData Data for submitting a claim
+    /// @dev The encoding of `_claimData` might vary depending on the history implementation
     /// @dev Should have access control.
+    ///      Incorrect claim indices could raise two errors:
+    ///      `InvalidInputIndices` if first index is posterior than last index or
+    ///      `UnclaimedInputs` if first index is not the subsequent of previous claimed index or
+    ///                        if the first index of the first claim is not zero.
```

Comment:
This is intrinsically tied to the implementation. I think we should move this to the implementation.